### ISSUE TITLE
Error in typings: Cannot find name 'Stream'.

### ISSUE THREAD

**Issue by gorillatron** [mostjs/core#286]
```
node_modules/@most/core/type-definitions/combinator/variadic.d.ts:6:19 - error TS2304: Cannot find name 'Stream'.

6   [K in keyof A]: Stream<A[K]>
                    ~~~~~~


Found 1 error.
```

**Comment by briancavalier** [comment]
Hey @gorillatron, thanks!  Fix coming up.

**Comment by briancavalier** [comment]
Fix released in `@most/core` 1.4.1.  Thanks again for reporting this!
